tool_quality_checks/quality_checks.py

```diff
@@ -28,6 +30,7 @@ def initialize_db_checks(db):
     """ Initialize database for checks """
 
     db.create_schema(schema_name="chk")
+    db.create_schema(schema_name="back")
```

Comment:
Wat is hiet idee van "back"? Tijdelijke backup? Iets wat je terug aan het zetten bent?
Het klinkt als een afko. Misschien een iets vollediger naam (tenzij het voor de rest voor iedereen duidelijk is).